translation-canary/translation_canary/translatable/__init__.py

```diff
@@ -36,14 +36,17 @@
 
 # Gather tests from this directory
 import pkgutil
+import importlib
```

Comment:
These changes needs to go to https://github.com/rhinstaller/translation-canary, the translation-canary folder here is just a git subtree and we want this fix for other projects that use it too.

Reply:
Done: https://github.com/rhinstaller/translation-canary/pull/7

Reply:
Thanks!